README.md

```diff
@@ -74,6 +74,7 @@ The table of error codes and their description:
 |Lacking long break             |LLB | week::Int, worker::String                                     |
 |Worker undertime hours         |WUH | hours::Int, worker::String                                    |
 |Worker overtime hours          |WOH | hours::Int, worker::String                                    |
+|Multiple teams simultaneously  |MWT | day::Int, from::Int, to::Int, teams::Vector{teams_names}      |
```

Comment:
Co być powiedział na `Workser teams collision`

Reply:
nie kminie from i to, to jest jakby jeden segment? Nie powinno być team1, team2 i vector segmentów no i dodatkowo dzien?

Reply:
No zamysł tutaj był taki, że ten błąd jest jako pojedyńczy segment, gdybyśmy mieli `>2` zespoły kiedyś bo dalej nie do końca kminię tego xD. Jeśli faktycznie będziemy mieli tylko 2 zespoły to można to nawet pominąć i zwracać wektor segmentów jak z np. `AON`. 

Do pomysłu z raportowaniem poszczególnych pracowników nie widziałem sensownego podejścia, więc odrzuciłem na rzecz zespołów.

Reply:
najnowsze info mówi, że mamy 4 zespoły

dogadam to, jak bedziemy te błędy pokazywać

Reply:
Jeśli mamy 4 zespoły, to wydaje mi się chyba najrozsądniejsze raportowanie pojedyńczych przedziałów wraz z zespołami jakie wtedy się spotykają. Agregacja segmentów dnia do jednego błędu byłaby nieczytelna

Reply:
kluczem będzie kombinacja zespołów, a wartością wektor przedziałów czasowych? why not? bądźmy konsekwentni, tak robimy dla pielęgniarek, imo similar case

Reply:
Czyli chcemy zwracać sumę wszystkich zespołów jakie nachodziły na siebie w ciągu całego dnia jako jedno?